tests/test_model.py

```diff
@@ -58,11 +57,11 @@ def test_cpes():
     ps2 = ProductStreamFactory(name="Ansercaerulescens", products=p1, productversions=pv2)
     ps3 = ProductStreamFactory(
         name="Brantabernicla",
-        cpe="cpe:/o:redhat:Brantabernicla:8",
         description="The brant or brent goose is a small goose of the genus Branta.",
         products=p1,
         productversions=pv2,
     )
+    pvariant1 = ProductVariantFactory(cpe="cpe:/o:redhat:Brantabernicla:8")
```

Comment:
Let's link the variant to the parent product 1 / version 2 / stream 3 for RHEL 8, then create a second variant with a different CPE, and link it to the parent product 1 / version 1 / stream 1 for RHEL 7:
```suggestion
    pvariant1 = ProductVariantFactory(cpe="cpe:/o:redhat:enterprise_linux:7", products=p1, productversions=pv1, productstreams=ps1)
    pvariant2 = ProductVariantFactory(cpe="cpe:/o:redhat:Brantabernicla:8", products=p1, productversions=pv2, productstreams=ps3)
```
If we do that, I think the old asserts will keep passing.